doc/architecture/20170816-audit-plan-to-model-audit-data.md

```diff
@@ -0,0 +1,29 @@
+
+
+## Term definitions
+
+A **theme** is a high level grouping criteria for all the content that GOV.Uk is currently hosting.
+
+An **inventory plan** is a set of rules that define the content that belongs to a theme. 
+
+The **inventory sheet** refines an inventory plan by excluding formats by adding/removing sets of content items.
```

Comment:
I think if we adopt the definition of an Inventory Plan I have proposed (see above), then the Inventory Sheet doesn't need to exist. We move away from the concept of a "sheet", and move towards the concept of:

1. Apply Inventory Plan to content, to assign everything to a Theme
1. Move Content Items in/out/between Themes (this conceptually doesn't need to be a "sheet")
1. Pick a subset of a Theme to audit - this is the Audit Sheet